packages/vscode-extension/src/debug/prerequisitesHandler.ts

```diff
@@ -154,6 +155,63 @@ async function checkPort(
   };
 }
 
+export async function checkPrerequisitesForGetStarted(): Promise<Result<any, FxError>> {
+  try {
+    try {
+      ExtTelemetry.sendTelemetryEvent(TelemetryEvent.GetStartedPrerequisitesStart);
+    } catch {
+      // ignore telemetry error
+    }
+
+    // node, account
+    const totalSteps = 2;
+    let currentStep = 1;
+    VsCodeLogInstance.outputChannel.show();
+    VsCodeLogInstance.info("Prerequisites Check");
+    VsCodeLogInstance.outputChannel.appendLine(
+      doctorConstant.CheckNumber.split("@number").join(`${totalSteps}`)
+    );
+
+    // Get project settings
+    const workspacePath = ext.workspaceUri.fsPath;
+    const localEnvManager = new LocalEnvManager(
+      VsCodeLogInstance,
+      ExtTelemetry.reporter,
+      VS_CODE_UI
+    );
+    const projectSettings = await localEnvManager.getProjectSettings(workspacePath);
```

Comment:
One scenario is - this may be called without any project, so need to handle empty project as well

Reply:
Added.
In that case, we will check azure node.